app/views/spree/taxons/_taxon.html.erb

```diff
@@ -1,5 +1,7 @@
-<h5>
-  <%= link_to taxon.name, seo_url(taxon), class: 'breadcrumbs' %>
-</h5>
+<section class="taxon">
```

Comment:
We should move this whole file (taxons/_taxon.html.erb) into a separate component.